portfolio.html

```diff
@@ -8,6 +8,284 @@
+                    </div>
+                    </div>
+                </div>
+
+            </div>
+
+        <div class="desktop-card-work-row-2">
+          <div class="desktop-card-1">
+            <img src="./images/mobile_version/Card/Img_Plaholder.png" class="desktop-image-cover" alt="image">
+            <div class="desktop-text-card-2">
+                <h2 class="desktop-card-1-heading">Multi-Post Stories <br> Gain+Glory</h2>
+            <ul class="desktop-list-languages">
+                <li class="ruby">Ruby on rails</li>
+                <li class="css">css</li>
+                <li class="javascript">JavScript</li>
+                <li class="html">html</li>
+            </ul>
+            <div class="desktop-button-frame">
+                <button type="button" class="desktop-btn-green">See Project</button>
+            </div>
+            </div>
+           </div>
+
+           <div class="desktop-card-1">
+            <img src="./images/mobile_version/Card/Img_Plaholder.png" class="desktop-image-cover" alt="image">
+            <div class="desktop-text-card-2">
+                <h2 class="desktop-card-1-heading">Multi-Post Stories <br> Gain+Glory</h2>
+            <ul class="desktop-list-languages">
+                <li class="ruby">Ruby on rails</li>
+                <li class="css">css</li>
+                <li class="javascript">JavScript</li>
+                <li class="html">html</li>
+            </ul>
+            <div class="desktop-button-frame">
+                <button type="button" class="desktop-btn-green">See Project</button>
+            </div>
+            </div>
+           </div>
+
+           <div class="desktop-card-1">
+            <img src="./images/mobile_version/Card/Img_Plaholder.png" class="desktop-image-cover" alt="image">
+            <div class="desktop-text-card-2">
+                <h2 class="desktop-card-1-heading">Multi-Post Stories <br> Gain+Glory</h2>
+            <ul class="desktop-list-languages">
+                <li class="ruby">Ruby on rails</li>
+                <li class="css">css</li>
+                <li class="javascript">JavScript</li>
+                <li class="html">html</li>
+            </ul>
+            <div class="desktop-button-frame">
+                <button type="button" class="desktop-btn-green">See Project</button>
+            </div>
+            </div>
+           </div>
+
+        </div>
+    </div>
+    </section>
+
+
+    <section class="desktop-about-me">
+        <img src="./images/desktop_version/frame_headline/illustration-about-me-2-desktop@2x.png" class="illustration-about-me-2" alt="Illustration icon-2">
+        <img src="./images/desktop_version/frame_headline/illustration-about-me-1-desktop@2x.png" class="illustration-about-me-1" alt="Illustration icon-1">
+        <div class="desktop-about-me-header">
+            <h2 class="desktop-about-me-primary-text">About me</h2>
```

Comment:
- [ ] You have used position absolute to center the elements, which is not recommended, kindly use flexbox or grid instead
![image](https://user-images.githubusercontent.com/62705947/186937492-5a304e74-2b20-4dbb-856f-8c1da8a3e2d6.png)